src/microbots/environment/Environment.py

```diff
@@ -21,3 +21,11 @@ def stop(self):
     @abstractmethod
     def execute(self, command: str, timeout: Optional[int] = 300) -> CmdReturn:
         pass
+
+    @abstractmethod
```

Comment:
Can we make `copy_to_container` and `copy_from_container` as optional methods to `Environment` Abstract class. In-case if the user implements his own environment for `ReadingBot` it may not required `copy_to_container` implementation in his environment